src/resolve/resolved-dns-query.h

```diff
@@ -95,6 +95,7 @@ struct DnsQuery {
         union in_addr_union request_address;
         unsigned block_all_complete;
         char *request_address_string;
+        char *request_name;
```

Comment:
please don't add this. the same info is available in `dns_question_first_name(q->question_utf8)`.